inst/extdata/PKAnalyses.csv

```diff
@@ -0,0 +1,15 @@
+﻿"IndividualId","QuantityPath","Parameter","Value","Unit"
```

Comment:
I also moved PKAnalyses.csv and SAResult.csv as these are required in the vignettes, too.